File: Java/Watermelon.java
import java.io.*;

public class Watermelon {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

    public static void main(String[] args) throws Exception {

        System.out.print("Ingrese el peso de la sandia: ");
        float peso = Integer.parseInt(br.readLine());

        System.out.println((peso % 2 == 0 && peso > 2) ? "Si" : "No");

    }
}

// https://omegaup.com/arena/problem/Watermel/

File: C++/Watermelon.cpp
#include <iostream>

using namespace std;

int main()
{
    int peso;
    cout << "Ingrese el peso de la sandia: ";
    cin >> peso;

    cout << (peso % 2 == 0 && peso > 2) ? "si" : "no";

    return 0;
}

// https://omegaup.com/arena/problem/Watermel/